modules/client-configuration/pages/supported-features-centos.adoc

```diff
@@ -24,7 +24,7 @@ The icons in this table indicate:
 .Supported Features on {centos} Operating Systems
 |===
 | Feature | {centos}{nbsp}6 | {centos}{nbsp}7 | {centos}{nbsp}8
```

Comment:
This would be easier to review with a render. Did I ever mention we should do it automatically on the PRs? :-D

Assuming it's OK :-)

Reply:
You might have mentioned it :wink:  I think @jcayouette and I keep vetoing the idea